### ISSUE TITLE
audio reconstruction

### ISSUE THREAD

**Issue by wafer9** [microsoft/VibeVoice#189]
I am using the VAE architecture from VibeVoice and training it from scratch for audio reconstruction. The training data is Emilia (100k hours).

The loss converges normally. The reconstruction loss is computed as:

recon_loss = torch.nn.functional.mse_loss(x.float(), y.float(), reduction="mean")

The value of recon_loss is around 0.001–0.002. However, the reconstructed audio spectrogram looks abnormal, as shown below.

What could be the possible reasons for this issue? @YaoyaoChang 

<img width="2412" height="712" alt="Image" src="https://github.com/user-attachments/assets/8089e044-c716-40d0-9df3-c3e1cbbf8988" />
